README.md

```diff
@@ -1,15 +1,15 @@
 # dogecoin-bootstrap
 
-## 2020-01-29: Latest torrent file for a up to date chain (uncompressed) available at https://dogecoin.gg/dogecoin-bootstrap-2021-01-29.torrent (Hosted/Maintained by https://github.com/IncognitoJam/IncognitoJam )
+## 2021-05-09: Latest torrent file for a up to date chain (uncompressed) available at https://dogecoin.gg/files/dogecoin-bootstrap-2021-05-09.torrent (Hosted/Maintained by https://github.com/IncognitoJam/IncognitoJam)
```

Comment:
```suggestion
## 2021-05-09: Latest torrent file for a up to date chain (uncompressed) available at https://dogecoin.gg/files/dogecoin-bootstrap-2021-05-09.torrent (Hosted/Maintained by https://github.com/IncognitoJam/IncognitoJam )
```
The space needs to stay, otherwise MD parsers might include it with the URL